datastore/gcloud/aio/datastore/datastore.py

```diff
@@ -346,3 +346,51 @@ def make_mutation(operation: Operation, key: Key,
         transaction = await self.beginTransaction(session=session)
         mutation = self.make_mutation(operation, key, properties=properties)
         await self.commit([mutation], transaction=transaction, session=session)
+
+    async def export(self, output_bucket_name: str,
+                     kinds: Optional[List[str]] = None, namespaces: Optional[List[str]] = None,
+                     labels: Optional[Dict[str, str]]= None,
+                     session: aiohttp.ClientSession = None, timeout: int = 10) -> Dict[str, str]:
+        project = await self.project()
+        url = f'{API_ROOT}/projects/{project}:export'
+
+        export_request = {
+            'entityFilter': {'kinds': kinds or [], 'namespaceIds': namespaces or []},
+            'outputUrlPrefix': f'gs://{output_bucket_name}'}
+        if labels:
+            export_request['labels'] = labels
```

Comment:
Would it be cleaner to do this the same way as the other values, eg. by unconditionally setting `'labels': labels or {}`, or does that break their API?

Reply:
updated